### PR TITLE
OWA-55: Make upload OWA UI responsive on smaller screens

### PR DESCRIPTION
**Pull request by jomadoye**
# JIRA TICKET NAME:
OWA-55: [Make upload OWA UI responsive on smaller screens](https://issues.openmrs.org/browse/OWA-55)

## SUMMARY:
This PR make upload OWA UI responsive on smaller screens

### CODE REVIEW

File: app/js/components/homePage/manageApps/AddAddon.jsx [hunk 1]
```diff
@@ -14,29 +14,37 @@ export default function AddAddon({ handleClear, handleUpload }) {
...
-            className="btn btn-primary"
-            onClick={handleUpload}>
-            <span 
-              className="glyphicon glyphicon-upload" /> Upload</button>
-          <button
-            className="btn btn-danger"
-            onClick={handleClear}>
-            <span 
-              className="glyphicon glyphicon-remove" /> Clear Addon</button>
+        <div className="row">
```
Comment (kingisaac95) [inline app/js/components/homePage/manageApps/AddAddon.jsx:17]: Nice Job

File: app/js/components/homePage/manageApps/AddAddon.jsx [hunk 2]
```diff
@@ -14,29 +14,37 @@ export default function AddAddon({ handleClear, handleUpload }) {
...
+          <div className="col-xs-12">
+            <div className="col-xs-6 col-sm-8 col-md-8 col-lg-8">
+              <input 
+                name="file"
+                type="file" 
+                id="fileInput"
+                accept=".zip"
```
Comment (kingisaac95) [inline app/js/components/homePage/manageApps/AddAddon.jsx:24]: Does addons come only as `.zip` files?

Comment (jomadoye) [reply app/js/components/homePage/manageApps/AddAddon.jsx:24]: Yes @kingisaac95, that's the only extension the addon-manager supports now, you can create a ticket to look into supporting other compressed files extension types like .RAR, .ARJ, .TAR, .GZ, and .TGZ. &#x1F60A;

**Comment by wanjikum** [comment]
Nice work Jedidah 😄

**Pull request merged**